.circleci/config.yml

```diff
@@ -208,7 +284,8 @@ jobs:
           paths:
             - /opt/circleci/.pyenv
             - ~/.torch
-          key: cache-{{ arch }}-<< parameters.pytorch_version >>-{{ .Branch }}-20210827
+          key: cache-{{ arch }}-<< parameters.pytorch_version >>-{{ .Branch }}-20990004
+
```

Comment:
There're a lot of duplication between `linux_cuda111_tests`, `linux_cuda113_tests` and `linux_cuda117_tests`, it would be nice to make it a bit more concise.

Reply:
I am not really familiar with CircleCI. I've just winged based on sampels. Any ideas on how to templatize even more?
The `- <<:` construction is already an abstraction layer, but maybe we can group them in a larger template if CircleCI has such mechanism